lib/cluster.js

```diff
@@ -21,5 +21,11 @@
 
 'use strict';
 
-const childOrPrimary = 'NODE_UNIQUE_ID' in process.env ? 'child' : 'primary';
+const {
+  privateSymbols: {
+    env_private_symbol,
+  },
+} = internalBinding('util');
+
+const childOrPrimary = 'NODE_UNIQUE_ID' in process[env_private_symbol] ? 'child' : 'primary';
```

Comment:
you may want to use ObjectHasOwn for this, since potentially anyone could add `NODE_UNIQUE_ID` to the prototype chain.
```suggestion
const childOrPrimary = ObjectHasOwn(process[env_private_symbol], 'NODE_UNIQUE_ID') ? 'child' : 'primary';
```